Automattic/wc-calypso-bridge
# eCommerce Signup Flow: Add the "Welcome" note back

⚠️ Please do not merge until June 7th, 2024 ⚠️ 

---

### Changes proposed in this Pull Request:

Related to https://github.com/Automattic/dotcom-forge/issues/7220.

* eCommerce Signup Flow: Add the "Welcome" note back

![331631991-a4fbe1d7-386c-4ffb-9b12-e96442d0a532](https://github.com/Automattic/wc-calypso-bridge/assets/25105483/6d12e7d7-3581-48ca-95f6-cb00e5479c45)






### How to test the changes in this Pull Request:




The changes proposed in this PR are targeting Entrepreneur Trial plan, however, due to technical limitations, we will only perform a partial test on a "full" Entrepreneur plan site.

If you don't have your WoA test site yet, please follow the steps outlined in pdDOJh-3ob-p2 and create a test site with Entrepreneur plan (the plan can be added through the SA).

1. Check out the PR and sync it with your WoA test site on the Entrepreneur plan.
2. Revert the last commit to apply the temporary test code: `git revert fd855cae32da54f4bb4e7209e368fecd1892fe57`
3. Navigate to `/wp-admin/admin.php?page=wc-admin` and review the Welcome note. It should include correct copy - as can be seen in the screenshot above.



### Other information:




### FOR PR REVIEWER ONLY:
